src/panels/BruinPanel.ts

```diff
@@ -399,6 +401,19 @@ export class BruinPanel {
             patchAssetCommand(assetData, this._lastRenderedDocumentUri);
             break;
 
+          case "bruin.fillAssetDependency":
+            if (!this._lastRenderedDocumentUri) {
+              console.error("No active document to fill asset dependency.");
+              return;
+            }
+            
+            const assetPath = this._lastRenderedDocumentUri.fsPath;
+            const assetWorkspaceDir = await bruinWorkspaceDirectory(assetPath);
+            
+            const command = `bruin patch fill-asset-dependencies "${assetPath}"`;
+            await runBruinCommandInIntegratedTerminal(command, assetWorkspaceDir);
```

Comment:
this is not the right interface, this command should accept an array of strings, and no `bruin`:
```suggestion
            const command = ["patch", "fill-asset-dependencies", "${assetPath}"];
            await runBruinCommandInIntegratedTerminal(command, assetWorkspaceDir);
```

Reply:
using an array isn't required in this case, as `sendText` accepts a string and passes it directly to the terminal. A plain string should work without issues.